src/screens/Tests/Tests.js

```diff
@@ -4,50 +4,289 @@ import { TestIntro, TestQuestion, BackButton, TestResults, SubmitTestButton, Pag
+		answear1: {
+			value: 'draggable',
+			correct: true
+		},
+		answear2: {
+			value: 'drag',
+			correct: false
+		},
+		answear3: {
+			value: 'context',
+			correct: false
+		}
+	},
+	{
+		question: 'Which of the following attribute triggers events when a form changes?',
+		answear1: {
+			value: 'onformchange',
 			correct: true
 		},
+		answear2: {
+			value: 'onchange',
+			correct: false
+		},
 		answear3: {
-			value: 'odp 3',
+			value: 'onedit',
 			correct: false
 		}
 	},
 	{
-		question: 'pytanko 2',
+		question: 'Which of the following tag represents an independent piece of content of a document in HTML5?',
 		answear1: {
-			value: 'odp 1.2',
+			value: 'section',
 			correct: false
 		},
 		answear2: {
-			value: 'odp 2.2',
+			value: 'article',
+			correct: true
+		},
+		answear3: {
+			value: 'aside',
+			correct: false
+		}
+	},
+	{
+		question: 'HTML tags are case sensitive.',
+		answear1: {
+			value: 'false',
+			correct: true
+		},
+		answear2: {
+			value: 'true',
 			correct: false
 		},
 		answear3: {
-			value: 'odp 3.2',
+			value: 'sometimes',
+			correct: false
+		}
+		
+	},
+	{
+		question: 'DOM stands for',
+		answear1: {
+			value: 'Document object model',
 			correct: true
+		},
+		answear2: {
+			value: 'Data object model',
+			correct: false
+		},
+		answear3: {
+			value: 'Data oriented model',
+			correct: false
 		}
 	},
 	{
-		question: 'pytanko 3',
+		question: 'Ancestor of all other elements on the page is called',
 		answear1: {
-			value: 'odp 1.3',
+			value: 'sibling',
+			correct: false
+		},
+		answear2: {
+			value: 'ancestor',
+			correct: false
+		},
+		answear3: {
+			value: 'parent',
 			correct: true
+		}
+	},
+	{
+		question: 'The sessionStorage object stores the data for',
+		answear1: {
+			value: 'all sessions',
+			correct: false
 		},
 		answear2: {
-			value: 'odp 2.3',
+			value: 'one session',
+			correct: true
+		},
+		answear3: {
+			value: '3 sessions in a row',
 			correct: false
+		}
+	},
+	{
+		question: 'The __________ is an instruction to the web browser about what version of HTML the page is written in',
+		answear1: {
+			value: '< DOCTYPE>',
+			correct: false
+		},
+		answear2: {
+			value: '< !DOCTYPE>',
+			correct: true
 		},
 		answear3: {
-			value: 'odp 3.3',
+			value: '< !TYPE>',
 			correct: false
 		}
-	}
+	},
+	{
+		question: 'What HTML stands for?',
+		answear1: {
+			value: 'Hyper Text Markup Language',
+			correct: true
+		},
+		answear2: {
+			value: 'Hyper Text Markdown Language',
+			correct: false
+		},
+		answear3: {
+			value: 'Home Tool Markup Language',
+			correct: false
+		}
+	},
+	{
+		question: 'What is the correct HTML for creating a hyperlink?',
+		answear1: {
+			value: '<a>http://www.intervyou.essa</a>',
+			correct: false
+		},
+		answear2: {
+			value: '<a href="http://www.intervyou.essa">IntervYou</a>',
+			correct: true
+		},
+		answear3: {
+			value: '<a url="http://www.intervyou.essa">IntervYou.essa</a>',
+			correct: false
```

Comment:
intervYou.essa ? I dnt recognize this domain name, regional on any neighborhood in Gda? 😃